src/settings/use-settings-defaults.ts

```diff
@@ -0,0 +1,4 @@
+export const DEFAULT_SETTING_SHOW_CHAT = true;
```

Comment:
This file name should be changes as well, imo.

Reply:
renamed @ f908a1489cfee5b6d593c45708b24e78493e99b5